static/app/utils/feedback/list/types.tsx

```diff
@@ -1,25 +1,23 @@
-import {
-  FeedbackItemResponse,
-  HydratedFeedbackItem,
-} from 'sentry/utils/feedback/item/types';
+import type {BaseGroup, GroupStats} from 'sentry/types';
 
-export type FeedbackListResponse = FeedbackItemResponse[];
```

Comment:
the types are different now, we can't share.

The list type has fewer fields compared to when we fetch the issue independently.